common/utils/dates.test.ts

```diff
@@ -29,13 +29,51 @@ describe('isSameDay', () => {
     expect(result).toEqual(true);
   });
 
-  it('says two times on the same day are the same', () => {
-    const result = isSameDay(
-      new Date(2001, 1, 1, 1, 1, 1),
-      new Date(2001, 1, 1, 13, 24, 37)
+  describe('ComparisonMode', () => {
+    const september19Midnight = new Date(
+      'Mon Sep 19 2022 00:00:00 GMT+0100 (British Summer Time)'
+    );
+    const september18TwentyThreeThirty = new Date(
+      'Sun Sep 18 2022 23:30:00 GMT+0100 (British Summer Time)'
+    );
+    const september19MidnightThirty = new Date(
+      'Mon Sep 19 2022 00:30:00 GMT+0100 (British Summer Time)'
+    );
+    const september19Midday = new Date(
+      'Mon Sep 19 2022 12:00:00 GMT+0100 (British Summer Time)'
     );
```

Comment:
Let's sprinkle in the UTC times as comments to help future readers.

```suggestion
    const september19Midnight = new Date(
      // aka Sun Sep 18 2022 23:00:00 UTC
      'Mon Sep 19 2022 00:00:00 GMT+0100 (British Summer Time)'
    );
    const september18TwentyThreeThirty = new Date(
      // aka Sun Sep 18 2022 22:30:00 UTC
      'Sun Sep 18 2022 23:30:00 GMT+0100 (British Summer Time)'
    );
    const september19MidnightThirty = new Date(
      // aka Sun Sep 18 2022 23:30:00 UTC
      'Mon Sep 19 2022 00:30:00 GMT+0100 (British Summer Time)'
    );
    const september19Midday = new Date(
      // aka Sun Sep 18 2022 11:00:00 UTC
     'Mon Sep 19 2022 12:00:00 GMT+0100 (British Summer Time)'
    );
```